packages/integration/src/github/GithubAppCredentialsMux.ts

```diff
@@ -0,0 +1,214 @@
+/*
```

Comment:
This is a lift and shift. No actual change expected.

Reply:
Tbh I think the easiest thing to do here would be to keep it as is so that we don't need to shift over the tests. It's worth deprecating the export of `GithubAppCredentialsMux` (later), since the reason it's currently being exported is just to be used as a utility for a rest call, which we can easily replace. If we do this split I think we should move over the tests as well, and that's just a bunch of work 😅